### DISCUSSION TITLE
KVM not listing Unmanaged Instances

### DISCUSSION THREAD

**Discussion by sghazra21** [apache/cloudstack#9156] [Q&A] [answered]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.19.0.1
~~~

##### CONFIGURATION

Advanced Networking

##### OS / ENVIRONMENT

OEL KVM libvirtd

##### SUMMARY

When using import-export tool for KVM Manage/unmanage Instance , Its listing the managed instances but its not listing any unmanaged instance which is created manually

##### STEPS TO REPRODUCE

1. Create a VM in KVM using virsh or virt-manager.
2. Try to list it in Import-Export tool on Cloudstack UI.

~~~

~~~

##### EXPECTED RESULTS

~~~
It should list VMs which are not created through cloudstack
~~~

##### ACTUAL RESULTS

~~~
Empty List
~~~

I want a method to onboard VMs on cloudstack.

**Comment by andrijapanicsb** [comment, marked as answer]
Please read the documentation. It lists's only stopped VMs and it requires listen_tcp=1 in the libvirtd config
